code/backend/batch/utilities/helpers/EnvHelper.py

```diff
@@ -79,10 +79,10 @@ def __load_config(self, **kwargs) -> None:
             "AZURE_SEARCH_USE_INTEGRATED_VECTORIZATION", "False"
         )
         self.AZURE_SEARCH_IV_MAX_PAGE_LENGTH = int(
-            os.getenv("AZURE_SEARCH_IV_MAX_PAGE_LENGTH", "2000")
+            os.getenv("AZURE_SEARCH_IV_MAX_PAGE_LENGTH", "800")
         )
         self.AZURE_SEARCH_IV_PAGE_OVERLAP_LENGTH = int(
-            os.getenv("AZURE_SEARCH_IV_PAGE_OVERLAP_LENGTH", "500")
```

Comment:
Can we also change the .env.sample (even though I'm unsure its used!) and in the LOCAL_DEPLOYMENT.md I dont think these new values have been added.

Reply:
I will update all the .md files, .env.sample in a separate PR. Will do it along with #761